dnload.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+#
+# Note to packagers:
+# Change DNLOAD_MODULE_PATH to the system directory when packaging dnload.
+#
+# For example:
+# setconf dnload.sh DNLOAD_MODULE_PATH=/usr/lib/python3.6/site-packages/dnload
+#
+
+DNLOAD_MODULE_PATH=dnload
+
+if [ -z $1 ]; then
+  echo -e '\n\e[94mdnload \e[92mr14\e[0m'
+  cat<<tac
+
+Provide the path to a C++ source file as the first argument.
```

Comment:
__main__.py itself contains extensive listing of all command line arguments that is displayed when invocation happens without any targets, and this seems to hide that. Would it be possible to just let that be displayed?

Of course you can add this short description to the help blurb in __main__.py.